### PR TITLE
refactor(Input): Change the logic of TextField input and rendering

### PR DESCRIPTION
**Pull request by capdiem**

### CODE REVIEW

File: src/Masa.Blazor/Components/FileInput/MFileInput.cs [hunk 1]
```diff
@@ -244,17 +242,17 @@ public async Task HandleOnFileChangeAsync(InputFileChangeEventArgs args)
                     files.Add(file);
                 }
 
-                await SetInternalValueAsync((TValue)files);
+                InternalValue = (TValue)files;
```
Comment (doddgu) [inline src/Masa.Blazor/Components/FileInput/MFileInput.cs:245]: This async method lacks 'await' operators and will run synchronously.

File: src/Masa.Blazor/Mixins/Selectable/MSelectable.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public async Task HandleOnChange()
 
         await ValidateAsync(true, input);
 
-        await SetInternalValueAsync(input);
+        InternalValue = input;
```
Comment (doddgu) [inline src/Masa.Blazor/Mixins/Selectable/MSelectable.cs:53]: Line 56, This async method lacks 'await' operators and will run synchronously.

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Masa.Blazor/Components/Slider/MSlider.cs [hunk 3]
```diff
@@ -359,7 +353,7 @@ public virtual async Task HandleOnSliderStartSwiping(EventTarget target, double
             OldValue = DoubleInternalValue;
             IsActive = true;
 
-            if (target.Class.Contains("m-slider__thumb-container"))
+            if ((target.Class ?? "").Contains("m-slider__thumb-container"))
```
[the remark this answers is not in the document]
Comment (capdiem) [reply src/Masa.Blazor/Components/Slider/MSlider.cs:356]: `target.Class?.Contains("m-slider__thumb-container") ?? false` 这样？

File: src/Masa.Blazor/Mixins/Selectable/MSelectable.cs [hunk 4]
```diff
@@ -48,9 +48,11 @@ public async Task HandleOnChange()
         var input = InternalValue;
         input = !input;
 
-        await ValidateAsync(true, input);
+        ForceValidate(input);
 
-        await SetInternalValueAsync(input);
+        InternalValue = input;
+
+        await Task.CompletedTask;
```
Comment (doddgu) [inline src/Masa.Blazor/Mixins/Selectable/MSelectable.cs:55]: Line 58 and 63, This async method lacks 'await' operators and will run synchronously

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**